rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/QrdaControllerV1.java

```diff
@@ -56,15 +60,26 @@ public QrdaControllerV1(final QrdaService qrdaService, final ValidationService v
 	 * Endpoint to transform an uploaded file into a valid or error json response
 	 *
 	 * @param file Uploaded file
+	 * @param test Whether the conversion is a test or not
 	 * @return Valid json or error json content
 	 */
 	@PostMapping(headers = {"Accept=" + Constants.V1_API_ACCEPT})
-	public ResponseEntity<String> uploadQrdaFile(@RequestParam MultipartFile file) {
+	public ResponseEntity<String> uploadQrdaFile(@RequestParam MultipartFile file,
+			@RequestHeader(required = false, name = "Purpose") String purpose) {
```

Comment:
Do we want to define an Enum of well-known purposes that we expect callers to use?

Reply:
I prefer the arbitrary string approach. It allows a specific message to be input - for example, if a bunch of conversions were happening due to a specific database migration, they could be marked as such in case rollbacks were needed.